packages/sdk/odahuflow/sdk/clients/model.py

```diff
@@ -32,16 +32,19 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def calculate_url(base_url: str, model_route: str = None, model_deployment: str = None,
-                  url_prefix: str = None, mr_client: ModelRouteClient = None):
+def calculate_url(base_url: str, token : str = None, model_route: str = None, model_deployment: str = None,
```

Comment:
To simplify the signature I suggest to pass an API client (`api_client: RemoteAPIClient`) instead of this full set of auth paramteres. Then you can use `ModelRouteClient.construct_from_other(api_client)`.
**BUT** currently `construct_from_other` only copies API URL and Token, I fixed that in this PR https://github.com/odahu/odahu-flow/pull/580, so you can merge my branch into yours and use new `construct_from_other` that copies all auth parameters.